backend/src/main/java/com/bakdata/conquery/util/VariableDefaultValue.java

```diff
@@ -0,0 +1,17 @@
+package com.bakdata.conquery.util;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+
+/**
+ * Marker annotation for the AutoDoc project to signal that this fields default value is not stable because it is e.g. randomly generated or the current time.
```

Comment:
Sollte das dann nicht ins Autodoc Projekt? kP wie die dependencies sind

Reply:
Ich gebe dir recht. Das sollten wir in einem Projekt-Refactor machen, in dem z.B. diese Annotation in einem extra Projekt ist, das von beiden (AutoDoc und Backend) importiert wird